doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -33,13 +33,12 @@ Traceability to Reference Model.
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `req.gen.ost.01` | Open source | The Architecture **must** use OpenStack APIs.|
 | `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through OpenStack APIs. |
-| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component.. |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. |
 | `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. |
-| `req.gen.cnt.03` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where the state is required it **must** be kept external to the component. |
 | `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven auto-scaling. |
-| `req.gen.rsl.01` | Resiliency | The Architecture **must** support resilient OpenStack components that are required for the continued operation of running workloads. |
+| `req.gen.rsl.01` | Resiliency | The Architecture **must** support resilient OpenStack components that are required for the continued availability of running workloads. |
 | `req.gen.rsl.02` | Resiliency | The Architecture **should** support resilient OpenStack service components that are are not subject to `req.gen.rsl.01`. |
-| `req.gen.avl.01` | Availability | The Architecture **must** provide High Availability for OpenStack components |
+| `req.gen.avl.01` | Availability | The Architecture **must** provide High Availability for OpenStack components. |
 
 
 <p align="center"><b>Table 2-1:</b> OpenStack General Requirements.</p>
```

Comment:
This is not "OpenStack General Requirements." but OpenStack based Reference Architecture general requirements.

Reply:
@CsatariGergely Are you suggesting we change the Section 2.3 Title from "OpenStack Requirements" to "Architecture and OpenStack Requirements"?